app/policies/article_policy.rb

```diff
@@ -0,0 +1,13 @@
+class ArticlePolicy < ApplicationPolicy
+  class Scope < Scope
+    
+    def index?
```

Comment:
still wrong... index and show must be placed outside Scope. Inside ArticlePolicy but outside Scope

Please read Pundit doc for as a refresher